cpp/src/CMakeLists.txt

```diff
@@ -28,7 +28,9 @@ if(BUILD_WITH_GLPK)
         toppra/solver/glpk-wrapper.cpp)
 endif()
 
-add_library(toppra SHARED ${SOURCE_FILES})
+add_library(toppra ${SOURCE_FILES})
+set_property(TARGET toppra PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
Does it do something when building shared libs ? By security, I would wrap it in an IF(NOT BUILD_SHARED_LIBS).

Reply:
It's fine to keep it when we build shared libs.

Reply:
From [CMake doc](https://cmake.org/cmake/help/latest/command/add_library.html):
> For SHARED and MODULE libraries the POSITION_INDEPENDENT_CODE target property is set to ON automatically.

So this is good. Sorry for the noise.